pkg/controller/resolvedofferings/region.go

```diff
@@ -0,0 +1,23 @@
+package resolvedofferings
+
+import (
+	"context"
+
+	v1 "github.com/acorn-io/runtime/pkg/apis/internal.acorn.io/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func AddDefaultRegion(ctx context.Context, c client.Client, app v1.AppInstance) error {
```

Comment:
```suggestion
func AddDefaultRegion(ctx context.Context, c client.Client, app *v1.AppInstance) error {
```

to make mutations this function makes to `app` fields stick.